client/components/common/NotificationCenter.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { TransitionGroup, Transition } from 'react-transition-group';
```

Comment:
Looking at the implementation here, I believe we can achieve the drop down effect with pure CSS Instead of using `react-transition-group`

Reply:
@tosmak16 I agree it makes sense to use CSS and we actually use CSS for the dropdown transition. But we have a click event to handle and also if we use CSS some transition elements will be present in the DOM directly when the components is mounted and we need the Transitioned component to appear only when called thus keeping our app lightweight

Reply:
@j33n well, I agree with what you said above partly but let's overlook this for now because of time. Great job! though. :+1: